runtime/service_al2_integ_test.go

```diff
@@ -0,0 +1,54 @@
+// +build al2
```

Comment:
Why do we need a separate test for AL2?  We should be running the same tests everywhere.